src/navigator_data_ingest/base/updated_document_actions.py

```diff
@@ -271,11 +271,55 @@ def parse(
     return [error for error in errors if error is not None]
 
 
+def update_field_in_all_occurences(
+    update: Tuple[str, Update],
+    update_config: UpdateConfig,
+) -> List[Union[str, None]]:
+    """Update the document slug in all occurences of the document in s3."""
+    # TODO Do we need to archive on slug updates? The reason for this was expensive
+    # translation and text extraction costs not slug updates?
+    # TODO This can be made more generic.
+
+    document_id, document_update = update
+    _LOGGER.info(
+        "Updating document field in all document occurences in s3.",
+        extra={
+            "props": {
+                "document_id": document_id,
+            }
+        },
+    )
+    errors = []
+    for prefix in [
```

Comment:
Opportunity for some soft documentation by adding list list to a variable called something like: `s3_document_occurences`, or `s3_prefixes_with_documents`, or something